src/runtime/pay-client-test.js

```diff
@@ -110,6 +110,9 @@ describes.realWin('PayClient', {}, (env) => {
   it('should initialize correctly', () => {
     expect(payClient.getType()).to.equal('PAYJS');
     expect(redirectVerifierHelperStubs.prepare).to.be.calledOnce;
+    const expectedResponse = Promise.resolve(INTEGR_DATA_OBJ);
+    payClient.handleResponse_(expectedResponse);
```

Comment:
Don't call this directly.  We want to test that this automatically gets called because it is bound correctly.
Possibly do something like on line 167?

Reply:
Discussed offline. Added tests to both experiment and non-experiment flow.